src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,164 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './NewMovie.scss';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    movie: {
+      title: '',
+      description: '',
+      imgUrl: '',
+      imdbUrl: '',
+      imdbId: '',
+    },
+    titleError: false,
+    imgUrlError: false,
+    imdbUrlError: false,
+    imdbIdError: false,
+  };
+
+  handlerChange = (event) => {
+    const { value, name } = event.target;
+
+    this.setState(state => ({ movie: {
+      ...state.movie,
+      [name]: value,
+    } }));
+
+    this.setState({ [`${name}Error`]: false });
+  }
+
+  // eslint-disable-next-line consistent-return
+  handlerSubmit= (event) => {
+    event.preventDefault();
+    // eslint-disable-next-line
+    const { title, imgUrl, imdbUrl, imdbId } = this.state.movie;
+
+    if (!title || !imgUrl || !imdbUrl || !imdbId) {
+      return this.setState({
```

Comment:
no need to return anything from the function

```suggestion
      this.setState({
```